mods/d2k/weapons/other.yaml

```diff
@@ -15,36 +15,36 @@ Sound:
 	Warhead@1Dam: SpreadDamage
 		Range: 0, 32
 		Falloff: 100, 100
-		Damage: 860
+		Damage: 1000
```

Comment:
I don't think tiny rounding errors really matter to the small D2k player base, so I'd prefer we stick with 860 and the old versus values since they're taken from the original game.